Atomic/util.py

```diff
@@ -119,7 +123,7 @@ def default_ro_container_context():
 
 def write_out(output, lf="\n"):
     sys.stdout.flush()
-    sys.stdout.write(str(output) + lf)
+    sys.stdout.write(output.encode('utf-8') + lf)
```

Comment:
Are you really sure this works on Python 3?

```
[walters@9845f984f09d ostree]$ python3
Python 3.4.3 (default, Mar 31 2016, 20:42:37) 
[GCC 5.3.1 20151207 (Red Hat 5.3.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> "foo".encode('utf-8') + "\n"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can't concat bytes to str
>>> 
```

i.e. don't we have to do `lf.encode('utf-8')` too?